@s-schweer Ah, looks like there is [one more](http://166.78.178.63:8080/job/PR/job/salt-pr-lint-n/400/violations/file/tests/unit/states/rabbitmq_user_test.py/) syntax error:
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/testing/tests/unit/states/rabbitmq_user_test.py", line 90
    changes = {'tags': {'new': ['user], 'old': 'user'}}
```

--- a/tests/unit/states/rabbitmq_user_test.py
+++ b/tests/unit/states/rabbitmq_user_test.py
@@ -87,7 +87,7 @@ class RabbitmqUserTestCase(TestCase):
                 self.assertDictEqual(rabbitmq_user.present(name, force=True),
                                      ret)
 
-                changes = {'tags': {'new': ['user], 'old': 'user'}}
+                changes = {'tags': {'new': ['user'], 'old': 'user'}}
                 ret.update({'changes': changes})
                 self.assertDictEqual(rabbitmq_user.present(name, tags=tag), ret)
 

